### PR TITLE
Add support for ipv6 addresses. Improve logging and env handling.

### PR DESCRIPTION
**Pull request by frankban**
Changes include:
- No longer focus on model uuids: remove the uuid option, and by default set jujuEnvUUID to an empty string (--config jujuEnvUUID:"" is no longer required). That config value is deprecated anyway and only used by old versions of gijoe. If debugging is such cases is required, the config can be easily overwritten  by using the usual -config ...
- Add support ipv6 controllers and the GUI being served from an ipv6 address.
- Include static files and config values logs in the server output, both with their own colors (yellow and pink).
- Fix nil pointer panic when using -nocolor.
- Complete the -env work with setting the right controller address and gisf to true. So now, you can run `guiproxy -env production` or `guiproxy -env staging` (or brian) without the need for adding any other config overrides (as the controller will be the right one, gisf is set to true and, as mentioned, jujuEnvUUID is always empty. The -config and -controller parameters can still be used to override the regular env setup.
- Simplify the env code.
- No longer serve the GUI from the root; use /gui instead. This is done to intercept errors related to assuming the GUI is served at the root path sooner rather than later. The / path redirects to /gui automatically.

### CODE REVIEW

**Review by makyo — APPROVED** [review]
LGTM. Thanks for the work on the URL path; I know I've run into that problem a couple of times, esp setting up the demo instance.

File: internal/guiconfig/config.go [hunk 1]
```diff
@@ -100,28 +102,54 @@ func ParseOverridesForEnv(env, v string) (map[string]interface{}, error) {
...
+// Environments holds a map of environment names to their corresponding info.
+var Environments = map[string]env{
+	"production": env{
+		ControllerAddr: "jimm.jujucharms.com:443",
+	},
+	"staging": env{
+		ControllerAddr: "jimm.staging.jujucharms.com:443",
+		overrides:      envOverrides("https://api.staging.jujucharms.com"),
+	},
+	"brian": env{
```
Comment (jcsackett) [inline internal/guiconfig/config.go]: While some of us know what this means, I think it's probably best to have this as an easter egg and maintain the "qa" option. :stuck_out_tongue:

Comment (jcsackett) [reply internal/guiconfig/config.go]: Mind you, I'm totally pro keeping `brian`. Just also add `qa`.

**Review by jcsackett — COMMENTED** [review]
This looks good to me. I'm not marking it as an official approval b/c my go knowledge is still tiny, so you should probably wait for @makyo or another go-fluent team member.

File: internal/guiconfig/config.go (hunk 1 above)
Comment (frankban) [reply internal/guiconfig/config.go]: Ok, changed to using the ugly ambiguous "qa", brian being a valid hidden option ;-)

**Comment by frankban** [comment]
Thank you both for the reviews!

**Pull request merged**